samples/server/main.go

```diff
@@ -0,0 +1,127 @@
+// Copyright 2019 The Go Cloud Development Kit Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Command server runs a simple HTTP server with integrated StackDriver tracing
```

Comment:
Still "Stackdriver". 🙂 

```suggestion
// Command server runs a simple HTTP server with integrated Stackdriver tracing
```